demos/apps/bookinfo/README.md

```diff
@@ -35,35 +35,46 @@ rename the tool to `istioctl`. For example:
 $ cp istioctl-osx /usr/local/bin/istioctl
 ```
 
-> Note: The following instructions assume your current working directory 
+> Note: The following instructions assume your current working directory
 > is the [istio repo root](https://github.com/istio/istio).
 
 ## Setup
 
-Execute the [Istio installation instructions](../../../kubernetes/README.md) 
+Execute the [Istio installation instructions](../../../kubernetes/README.md)
 to install the Istio manager, mixer, and an envoy-based ingress controller,
 which will be used to implement the gateway for the application.
 (Note: the current version of the bookinfo application MUST use the `default` Kubernetes namespace.)
 
 ## Running the Bookinfo Application
 
 1. Change your current working directory to the bookinfo application directory:
-   
+
    ```bash
    cd demos/apps/bookinfo
    ```
 
 1. Bring up the application containers:
 
    ```bash
-   $ kubectl create -f bookinfo-istio.yaml
+   $ source ../../../kubernetes/istioctl-kube-inject.sh
```

Comment:
Move this up before the previous step (cd) then it's only `source ./kubernetes/istioctl-kube-inject.sh`